src/listmode_buildblock/LmToProjDataBootstrap.cxx

```diff
@@ -227,12 +232,14 @@ start_new_time_frame(const unsigned int new_frame_num)
   num_times_to_replicate_iter = num_times_to_replicate.begin();
     
   info(boost::format("Filled in replication vector for %1% events.") % total_num_events_in_this_frame);
+//  this->lm_data_ptr->set_get_position(start_of_this_frame);
+  this->lm_data_ptr->reset();
```

Comment:
I'd rather use the `get_position`. this could slow us down quite a lot when doing multiple time frames

Reply:
Mh I see. I think my get_position for SPECT need to be fixed for this